include/exadg/compressible_navier_stokes/spatial_discretization/kernels_and_operators.h

```diff
@@ -1012,9 +1021,10 @@ class ViscousOperator
     scalar
     get_penalty_parameter(FaceIntegratorScalar & fe_eval_m, FaceIntegratorScalar & fe_eval_p) const
   {
-    scalar tau = std::max(fe_eval_m.read_cell_data(array_penalty_parameter),
-                          fe_eval_p.read_cell_data(array_penalty_parameter)) *
-                 IP::get_penalty_factor<Number>(degree, data.IP_factor) * nu;
+    scalar tau =
+      std::max(fe_eval_m.read_cell_data(array_penalty_parameter),
+               fe_eval_p.read_cell_data(array_penalty_parameter)) *
+      IP::get_penalty_factor<dim, Number>(degree, using_simplex(fe_eval_m), data.IP_factor) * nu;
```

Comment:
I do not like the interface with `using_simplex`, since this does not follow the "open-closed" principle (open for extensions, closed for modifications). We have `ExaDG::ElementType`, which should be passed here.